docs/API/API-socket-options.md

```diff
@@ -1068,20 +1070,25 @@ undefined behavior:
 
   - **Bidirectional in Caller-Listener arrangement**: it is recommended to use
   a rule whereby you will be setting the `PBKEYLEN` exclusively either on the
-  Listener or on the Caller. The value set on the Listener will win, if set on
-  both parties.
+  Listener (the service defines the encryption rules strictly) or on the Caller
+  (the service allows all clients to freely decide about encryption, or the
+  server uses more elaborate rules for encryption).
 
-  - **Bidirectional in Rendezvous arrangement**: you have to know the passphrases
-  for both parties, as well as `PBKEYLEN`. Set `PBKEYLEN` to the same value on
-  both parties (or leave the default value on both parties, which will
-  result in 16)
+  - **Bidirectional in Rendezvous arrangement**: In rendezvous there's no
+  situation of a potential "unknown party", as it is with listener side, so
+  both parties simply should set the same passphrase and the same key length,
+  or both should leave the `SRTO_PBKEYLEN` option unchanged (which results in
+  default 16).
 
   - **Unwanted behavior cases**: if both parties set `PBKEYLEN` and the value
-  on both sides is different, the effective `PBKEYLEN` will be the one that is
-  set on the Responder party, which may also override the `PBKEYLEN` 32 set by
-  the sender to value 16 if such value was used by the receiver. The Responder
-  party is the Listener in a Caller-Listener arrangement. In Rendezvous it's a
-  matter of luck which party becomes the Responder.
+  on both sides is different, this is considered a conflict. It is resolved by
+  the Initiator party, which takes its own value, if it is set `SRTO_SENDER`
+  option to true, or takes the value from the Responder party otherwise. The
+  assignment of Initiator-Responder roles matches the Caller-Listener layout, and
+  in case of Rendezvous it's the matter of luck resulting from cookie comparison.
+  **It is highly recommended to never allow this to happen**, as this may also
+  override one party's setting of length 32 by the other one's 16.
```

Comment:
```suggestion
  on both sides is different, this is considered a conflict. It is resolved by
  the Initiator party, which takes its own value, if the `SRTO_SENDER`
  option is set to true. Otherwise, it takes the value from the Responder party. The
  assignment of Initiator-Responder roles matches the Caller-Listener layout. In the case of Rendezvous the assignment is a matter of luck, resulting from the cookie comparison.
  **It is highly recommended to never allow this to happen**, as this may result in having
  one party's setting of length = 32 be overridden by the other party's setting of length = 16.
```